content/docs/next-release/troubleshooting.md

```diff
@@ -4,4 +4,68 @@ description: Solve commonly encountered issues
+
+    2018-12-10 17:06:30 INFO  Configuration:236 - Initialized  tracer=JaegerTracer(...,  reporter=CompositeReporter(reporters=[RemoteReporter(sender=HttpSender(),  ...), ...]), ...)
+
+*IMPORTANT* the Jaeger Java Client will not fail when a connection to the Jaeger Collector can't be established. Spans will be collected and placed in an internal buffer. They might eventually reach the collector once a connection is established, or get dropped in case the buffer reaches its maximum size.
+
+## Increase the logging in the backend components
+
+The Jaeger Agent and Collector provide useful debugging information when the log level is set to `debug`. Every UDP packet that is received by the agent is logged by the agent, as well as every batch that is sent by the agent to the collector. The collector also logs every batch it receives and logs every span that is stored in the permanent storage.
+
+Here's what to expect when the Jaeger Agent is started with the `--log-level=debug` flag:
+
+    {"level":"debug","ts":1544458854.5367086,"caller":"processors/thrift_processor.go:113","msg":"Span(s) received by the agent","bytes-received":359}
+    {"level":"debug","ts":1544458854.5408711,"caller":"tchannel/reporter.go:133","msg":"Span batch submitted by the agent","span-count":3}
+
+In the collector's side, these are the expected log entries when the flag `--log-level=debug` is specified:
+
+    {"level":"debug","ts":1544458854.5406284,"caller":"app/span_handler.go:90","msg":"Span batch processed by the collector.","ok":true}
+    {"level":"debug","ts":1544458854.5406587,"caller":"app/span_processor.go:105","msg":"Span written to the storage by the collector","trace-id":"e66dc77b8a1e813b","span-id":"6b39b9c18f8ef082"}
+    {"level":"debug","ts":1544458854.54068,"caller":"app/span_processor.go:105","msg":"Span written to the storage by the collector","trace-id":"e66dc77b8a1e813b","span-id":"d92976b6055e6779"}
+    {"level":"debug","ts":1544458854.5406942,"caller":"app/span_processor.go:105","msg":"Span written to the storage by the collector","trace-id":"e66dc77b8a1e813b","span-id":"a56f41e38ca449a4"}
+
+## Check the /metrics endpoint
+
+For the cases where it's not possible or desirable to increase the logging in the collector's side, the `/metrics` endpoint can be used to check if spans for specific services were received. Assuming that the Jaeger Collector is available under a host named `jaeger-collector`, here's a sample `curl` call to obtain the metrics:
+
+    curl http://jaeger-collector:14268/metrics
+
+The following metrics are of special interest:
+
+    jaeger_collector_spans_received
+    jaeger_collector_spans_saved_by_svc
+    jaeger_collector_traces_received
+    jaeger_collector_traces_saved_by_svc
+
+The first two metrics should have similar values for the same service. Similarly, the two `traces` metrics should also have similar values. For instance, this is an example of a sane setup:
```

Comment:
Sane?  

Is the example a healthy configuration?  A sensible configuration?  I'm not sure what you were trying to get across here.

Reply:
"of a setup that is working appropriately" ?

Reply:
Or rather, "working as expected".